amplify/function/BusinessLogic/AssignUsersToTeams/handler.ts

```diff
@@ -0,0 +1,104 @@
+import { Amplify } from "aws-amplify";
+import { generateClient } from "aws-amplify/data";
+import type { AppSyncResolverHandler } from "aws-lambda";
+
+import type { Schema } from "../../../data/resource";
+import { modelIntrospection } from "./amplifyconfiguration.json";
+
+const MAX_TEAM_MEMBERS = 6;
+Amplify.configure(
+  {
+    API: {
+      GraphQL: {
+        endpoint: process.env.AMPLIFY_DATA_GRAPHQL_ENDPOINT as string, // replace with your defineData name
+        region: process.env.AWS_REGION,
+        defaultAuthMode: "iam",
+        modelIntrospection: modelIntrospection as never,
+      },
+    },
+  },
+  {
+    Auth: {
+      credentialsProvider: {
+        // eslint-disable-next-line @typescript-eslint/require-await
+        getCredentialsAndIdentityId: async () => ({
+          credentials: {
+            accessKeyId: process.env.AWS_ACCESS_KEY_ID as string,
+            secretAccessKey: process.env.AWS_SECRET_ACCESS_KEY as string,
+            sessionToken: process.env.AWS_SESSION_TOKEN as string,
+          },
+        }),
+        clearCredentialsAndIdentityId: () => {
+          /* noop */
+        },
+      },
+    },
+  },
+);
+
+const dataClient = generateClient<Schema>({
+  authMode: "iam",
+});
+
+type ResolverArgs = { userId: string; teamId: string };
+
+type ResolverResult = {
+  body: { value: string };
+  statusCode: number;
+  headers: { "Content-Type": string };
+};
+
+export const handler: AppSyncResolverHandler<
+  ResolverArgs,
+  ResolverResult
+> = async (event, _) => {
+  let team;
+  try {
+    team = await dataClient.models.Team.get({ id: event.arguments.teamId });
+  } catch (err) {
+    return {
+      body: { value: "Error: Team does not exist" },
+      statusCode: 404,
+      headers: { "Content-Type": "application/json" },
+    };
+  }
```

Comment:
let's make this check be like the user one for two reasons:
1. This won't throw/catch if ID isn't found since GraphQL returns a successful promise with an errors:[] field.
2. Just to be consistent